src/text.ts

```diff
@@ -24,19 +25,41 @@ const getTwilioClient = () => {
   return client
 }
 
-export const sendText = async ({ body, to, logger }) => {
+export const sendTwilioText = async ({ body, to, logger }) => {
   try {
     await getTwilioClient().messages.create({
       from: twilioPhoneNumber,
       to,
       body,
     })
   } catch (err) {
-    logger.fatal({ err }, "impossible to send text")
+    logger.fatal({ err }, "impossible to send text through Twilio")
     return
   }
 
-  logger.info({ to }, "sent text successfully")
+  logger.info({ to }, "sent text through Twilio successfully")
+}
+
+export const sendSMSalaText = async({ body, to, logger }) => {
+  try {
+    const base_url = "http://api.smsala.com/api/SendSMS"
+    const api_id = process.env.SMSALA_API_ID
+    const api_password = process.env.SMSALA_API_PASSWORD
+    const sms_type = "T"
+    const encoding = "T"
+    const sender_id = process.env.SMSALA_SENDER_ID
+    const phoneNumber = to.replace(/\D/g,'')
```

Comment:
what is the goal of this regex?

@bodymindarts here is where I think a comment would be useful. As a comment minimalist do you agree with that? :)

Reply:
I [agree](https://xkcd.com/1171/)